lxd/include/lxd_seccomp.h

```diff
@@ -22,6 +22,9 @@
 
 #ifndef SECCOMP_USER_NOTIF_FLAG_CONTINUE
 #define SECCOMP_USER_NOTIF_FLAG_CONTINUE 0x00000001
+#else
+#undef SECCOMP_USER_NOTIF_FLAG_CONTINUE
+#define SECCOMP_USER_NOTIF_FLAG_CONTINUE 0x00000001
```

Comment:
Seems like this should have belonged to the previous PR, can you make it a separate commit?